app/templates/administrative-units/index.hbs

```diff
@@ -44,8 +44,8 @@
           </div>
           <div class="au-o-grid__item au-u-1-2@small au-u-1-1@medium">
             <AuLabel for="classification">Type bestuur</AuLabel>
-            <ClassificationSelect
-              @selected={{this.classificationId}}
+            <ClassificationMultiSelect
```

Comment:
To keep the components naming consistent between multi-selects, would it be possible to rename the component to `ClassificationMultipleSelect`?